genny/model/_fixtures/models/widget.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/gofrs/uuid"
 )
 
+// Widget ...
```

Comment:
I'm not sure it's a good idea to have such dummy comment. :) How about something more relevant, like `Widget is used by pop to map the widgets table.`?

Reply:
I agree. But I might make it something more along the lines of:
> Widget is used by pop to map your widgets database table to your go code.